### DISCUSSION TITLE
Automatically switched to the default avatar after changing to a custom avatar

### DISCUSSION THREAD

**Discussion by HYPE-STUDIOS-PHILIP** [mozilla/hubs#3434] [General Questions & Discussion] [answered]
Hi! Is hubs down? The lighting in scenes comes and goes, I can't add any new characters and it seams that its pretty buggy

**Comment by brianpeiris** [comment, marked as answer]
We haven't seen any server issues on our end. Are you still running into these problems? 
Also, just to be sure, I assume you're referring to hubs.mozilla.com and not your own Hubs Cloud instance, correct?
If you're still having issues, please describe them in more detail so that we can help troubleshoot.

**Reply by HYPE-STUDIOS-PHILIP** [reply]
Hi! The issues i was faceing seem to be gone, but I still can't add my new characters. I've made characters before without and problems but this time I can't get it to work. I've uploaded the .glb file and I can se the preview image of my character and can select him, but when i choose the character it only changes the avatar for a new default avatar. Can't figuer out why this is happening..

**Reply by brianpeiris** [reply]
It sounds like the GLB you've uploaded is invalid in some way. When we detect and invalid avatar, we switch you back to a default one. It might help if you opened the developer console in your browser and watched for any errors that appear when you try to use your avatar.

**Reply by robinkwilson** [reply]
Marking this as answer, unmark if you upload the new avatar for @cvanmeurs to look at !

**Comment by cvanmeurs** [comment]
If you want to post the avatar, I can take a look at it.
